src/ConnectedVMware/docs/New-AzConnectedVMwareVMGuestAgent.md

```diff
@@ -37,27 +37,18 @@ Create GuestAgent.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Enable Guest Agent on VM
 ```powershell
-{{ Add code here }}
+New-AzConnectedVMwareGuestAgent -Name "default" -VirtualMachineName "test-vm" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" -CredentialsUsername $VM_User_Name -CredentialsPassword $Secure_String_Pwd -ProvisioningAction "install"
```

Comment:
>-Name "default" -VirtualMachineName "test-vm" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" [](http://example.com/codeflow?start=32&length=129)

same here. These params are not defined in the syntax above. Do these work?

Reply:
updated